### PR TITLE
Landing/integrate graph

### PR DESCRIPTION
**Pull request by FrozenKiwi**
I could watch this simulator all day...

https://user-images.githubusercontent.com/4876160/152660129-763a88c2-5fe4-4496-b365-cf6855117109.mp4

### CODE REVIEW

File: apps/site-landing/src/containers/BenefitsSimulator/Graph/index.tsx [hunk 1]
```diff
@@ -1,71 +1,118 @@
...
-    }
+    <Segment>
+      <Dimmer active={isLoading}>
+        <GraphLoading progress={progress} />
+      </Dimmer>
+      <AreaGraph maxGraphPoints={maxGraphPoints} data={displayData} />
     </Segment>
-  );
+  )
+  // return isLoading
```
Comment (MarieTheCoin) [inline apps/site-landing/src/containers/BenefitsSimulator/Graph/index.tsx:106]: Can this be removed?

File: apps/site-landing/src/containers/ReturnProfile/data/params.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ type ShockAbsorber = {
 // The data we use to calculate a clients potential returns
 export type SimulationParameters = {
   // How many months should we be calculating this for?
-  maxDuration: DurationObject;
+  //maxDuration: DurationObject;
```
Comment (MarieTheCoin) [inline apps/site-landing/src/containers/ReturnProfile/data/params.ts:38]: Can this be removed?

**Review by MarieTheCoin — APPROVED** [review]
I created a PR to fix the styles for the form. It's a temp fix just in case and the mobile version needs love.

File: apps/site-landing/src/containers/BenefitsSimulator/Graph/BenefitsGraph.stories.tsx [hunk 3]
```diff
@@ -2,14 +2,17 @@ import React, { useEffect, useState } from 'react';
 import { Meta, Story } from '@storybook/react';
 import { BenefitsGraph as Component } from '.';
 import { createParams, MarketData } from '../../ReturnProfile/data';
+//@ts-ignore
```
[the remark this answers is not in the document]
Comment (FrozenKiwi) [reply apps/site-landing/src/containers/BenefitsSimulator/Graph/BenefitsGraph.stories.tsx:5]: yes/no.  It shows up as an error in VS Code, but it actually isn't (it's reading a source file outside of the normal directory, but that doesn't apply to stories).  The `//ts-ignore` just makes it so VSCode doesn't show the file as red

File: apps/site-landing/src/containers/BenefitsSimulator/Graph/index.tsx (hunk 1 above)
Comment (FrozenKiwi) [reply apps/site-landing/src/containers/BenefitsSimulator/Graph/index.tsx:106]: yes, done

File: apps/site-landing/src/containers/ReturnProfile/data/params.ts (hunk 2 above)
Comment (FrozenKiwi) [reply apps/site-landing/src/containers/ReturnProfile/data/params.ts:38]: yeah, of course.  It seems pretty unlikely we'll go back to the old way

**Comment by FrozenKiwi** [comment]
> I created a PR to fix the styles for the form. It's a temp fix just in case and the mobile version needs love.

You're a legend Marie :-).

If you don't mind, I'll approve and merge that PR immediately?  I'm currently working on adding a -lot- more parameters to the simulator, so it'd be better to have it in sooner rather than later.

**Pull request merged**